app/models/order_detail.rb

```diff
@@ -383,6 +385,14 @@ def self.ordered_or_reserved_in_range(start_date, end_date)
     end
   end
 
+  def reconcile_if_skip_review
+    should_change_status = actual_total &&
+                           complete? &&
+                           product.billing_mode == "Skip Review"
+
+    change_status!(OrderStatus.reconciled) if should_change_status
+  end
```

Comment:
I'd pull this out into a little method like:
```
def auto_reconcile?
  product.billing_mode == "Skip Review" &&
    complete? &&
    actual_total
end
```
...then 
```suggestion
  def reconcile_if_skip_review
    change_status!(OrderStatus.reconciled) if auto_reconcile?
  end
```